generic_petg.xml.fdm_material

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic PLA profile. Serves as an example file, data in this file is not correct.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>Generic</brand>
+            <material>PETG</material>
+            <color>Generic</color>
+        </name>
+        <GUID></GUID>
```

Comment:
The original author of these materials probably has a unique way of generating the numbers, So I do not want to fiddle with it.
Or we should make clear where to get a unique number.